app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -247,7 +247,7 @@ class CheckForWaitingPullRequests extends ApiRequestHandler<Body> {
           title: title,
           sha: sha,
           labelId: labelId!,
-          emptyValidations: checkRuns.isEmpty || statuses.isEmpty,
+          emptyChecks: checkRuns.isEmpty,
```

Comment:
This needs to be handled in a per repo basis. We don't want to allow empty statuses in all the repos. E.g. luci-status has to be present on engine and flutter

Reply:
Updated. I added this check explicitly in the _checkStatuses to ensure repos with tree status ensure the status is set